php/release/Dockerfile

```diff
@@ -0,0 +1,147 @@
+FROM gcr.io/gcp-runtimes/ubuntu_20_0_4
+ENV PHP_DIR=/opt/php81
+ENV PHP_SRC_DIR=/usr/local/src/php81-build
+ENV PATH=${PATH}:/usr/local/bin:${PHP_DIR}/bin
+ENV PHP_VERISON=8.1.18
```

Comment:
We might want to bump this, since 8.1 is currently receiving [security fixes only](https://www.php.net/supported-versions.php), and will be EOL January 2026.

We should be good to bump to PHP 8.4